backend/controller/payment.controller.js

```diff
@@ -179,8 +164,8 @@ const verifySubscription = asyncHandler(async (req, res, next) => {
 
 const createPlan = asyncHandler(async (req, res, next) => {
   const { planName, amount, description, active } = req.body;
-
-  if (!planName || !amount || !description || !active) {
+  console.log(req.body);
+  if (!planName || !amount || !description) {
     return next(
       new CustomError(
         "All fields are required. planName, amount, description, active"
```

Comment:
You are not checking active, then why send it in response?

Reply:
removed